src/vs/workbench/contrib/notebook/browser/notebook.contribution.ts

```diff
@@ -1026,8 +1026,8 @@ configurationRegistry.registerConfiguration({
 			type: 'boolean',
 			default: true
 		},
-		[NotebookSetting.findScope]: {
-			markdownDescription: nls.localize('notebook.findScope', "Customize the Find Widget behavior for searching within notebook cells. When both markup source and markup preview are enabled, the Find Widget will search either the source code or preview based on the current state of the cell."),
+		[NotebookSetting.findFilters]: {
```

Comment:
Let's not change this setting this time otherwise it breaks users' settings.

Reply:
This shouldn't effect users at all, as they will have the same name in both the settings UI and json. This just changes the internal reference name. Name is still defined at https://github.com/microsoft/vscode/blob/41e9cb0cd6d8dc60441bbaca3b350d98dfce4f04/src/vs/workbench/contrib/notebook/common/notebookCommon.ts#L956